test/appium/tests/atomic/chats/test_group_chat.py

```diff
@@ -0,0 +1,163 @@
+from faker import Faker
+
+from tests import marks, get_current_time
+from tests.base_test_case import MultipleDeviceTestCase
+from tests.users import chat_users
+from views.sign_in_view import SignInView
+
+
+def create_users(driver_1, driver_2, username_1=Faker().name(), username_2=Faker().name()):
+    device_1_sign_in, device_2_sign_in = SignInView(driver_1), SignInView(driver_2)
+    return device_1_sign_in.create_user(username=username_1), device_2_sign_in.create_user(username=username_2)
+
+
+def create_and_join_group_chat(device_1_home, device_2_home, errors, chat_name=Faker().name()):
+    # device 2: get public key
+    device_2_public_key = device_2_home.get_public_key()
+    profile_view = device_1_home.profile_button.click()
+    device_2_default_username = profile_view.default_username_text.text
+    device_2_home.home_button.click()
+
+    # device 1: add device 2 as contact
+    device_1_chat = device_1_home.add_contact(device_2_public_key)
```

Comment:
device1 stayed at profile screen (line 17) so it won't find '+' button to add a contact.

Reply:
it actually went back to home screen (line 19). But anyway, I will fix "profile_view.default_username_text.text belongs to device1, so wrong assigment device_2_default_username"

Reply:
on line 17 is ‘device_1_home’ but on line 19 ‘device_2_home’ went back to home screen